Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx

```diff
@@ -418,9 +428,8 @@ export default function ChannelPage(props: IProps) {
                                 BtnClass={'btn-info' }
```

Comment:
we probably should track loading state of the SpareList request and add a spinning wheel while that is loading too

Reply:
I can disable it while loading, but changing the button to a spinning wheel would require a gemstone change. Alternatively, I could add a loading screen to the whole page.

Reply:
disable is fine